solutionktlist1/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,28 @@
+fun main(args: Array<String>) {
+    var name:String?= null
+    name?.let{it
+    println("waad")}
+}
+
+    class School(){
+        lateinit var studentsNames:String
+        lateinit var teachersNames:String
+        lateinit var adminNames:String
+    }
+    fun main(){
+        var School:School?=null
+        println("School studentsNames:")
+        School?.run{
+            println(this.studentsNames)
+            var Dammam=School
+        }
```

Comment:
All I need is calling numbers function here, write `numbers()` here to calling it